plib/controllers/IndexController.php

```diff
@@ -12,17 +12,21 @@ public function init()
         // Init tabs for all actions
         $this->view->tabs = [
             [
-                'title' => 'Form',
+                'title' => pm_Locale::lmsg('formTitle'),
                 'action' => 'form',
             ],
             [
-                'title' => 'Tools',
+                'title' => pm_Locale::lmsg('toolsTitle'),
                 'action' => 'tools',
             ],
             [
-                'title' => 'List',
+                'title' => pm_Locale::lmsg('listTitle'),
                 'action' => 'list',
             ],
+            [
```

Comment:
ActiveList is absent in Plesk 12.5. So we shouldn't add the tab with ActiveList in case of Plesk 12.5. Use "if (plesk version >= 17)" conditional clause. See "ext-custom-button" extension for example.